cloud-functions/cf-emailer/src/main/java/org/broadinstitute/ddp/cf/AttachmentLoadedFunction.java

```diff
@@ -0,0 +1,141 @@
+package org.broadinstitute.ddp.cf;
+
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.functions.BackgroundFunction;
+import com.google.cloud.functions.Context;
+import com.google.cloud.storage.Blob;
+import com.google.cloud.storage.BlobId;
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+import com.sendgrid.*;
+import org.apache.http.impl.client.HttpClients;
+
+import java.io.IOException;
+import java.nio.channels.Channels;
+import java.util.Map;
+import java.util.logging.Logger;
+import java.util.regex.Pattern;
+
+@SuppressWarnings("unused")
+public class AttachmentLoadedFunction implements BackgroundFunction<AttachmentLoadedFunction.GcsEvent> {
+    private static final Logger logger = Logger.getLogger(AttachmentLoadedFunction.class.getName());
+    private static final String ATTACHMENT_DISPOSITION = "attachment";
+    private static final String ATTACHMENT_PDF_TYPE = "application/pdf";
+    public static final String PATH_MAIL_SEND = "mail/send";
+
+    private static final String API_KEY = "SG_KEY";
+    private static final String FROM_EMAIL = "FROM_EMAIL";
+    private static final String FROM_NAME = "DEV";
+    private static final String TO_EMAIL = "TO_EMAIL";
+    private static final String TEMPLATE_ID = "d-5ec2c969bbad4fffabb8168abafa470c";
+    private static final String GCP_PROJECT = "broad-ddp-dev";
```

Comment:
Again, these will go into the config file we're using for the study-contact CF.

Reply:
Config made